content/articles/Introduction-to-Netcat/index.md

```diff
@@ -0,0 +1,165 @@
+### Introduction
+Netcat is a network analysis tool developed for security specialists and hackers to analyze a network in traffic.
+
+It is considered a `swiss-army knife` in information technology as it 
+
+At its early stages, netcat was developed to open up UDP/TCP connections between two machines from any desired port.
+
+Over time, it has been improved and now it is used for port scanning, port forwarding, file transfer, and also remote administration. This article will help us understand how Netcat performs these operations.
+
+Netcat can further be used as a back-end device that can be used or driven by other programs or scripts.
+
+It is also a must-have tool for pen-tester since it is essential for network debugging and investigation.
+
+### Goals
+In this tutorial, we will mainly focus on the following:
+- [Executing general Netcat commands](#executing-commands-on-netcat)
+- [Modes of running Netcat](#modes-of-running-netcat)
+- [Uses of Netcat](#uses-of-netcat)
+
+### Executing commands on Netcat
+Netcat is only available in the command-line interface. You can access Netcat on the `command line` on Windows and `bash terminal` on Linux.
```

Comment:
Before getting into this, add the prerequisites and guide us on how we can install Netcat.